airbyte-webapp/src/components/common/MainPageWithScroll/MainPageWithScroll.tsx

```diff
@@ -13,17 +13,30 @@ interface MainPageWithScrollProps {
   headTitle?: React.ReactNode;
   pageTitle?: React.ReactNode;
   children?: React.ReactNode;
+  noBottomPadding?: boolean;
 }
 
-export const MainPageWithScroll: React.FC<MainPageWithScrollProps> = ({ headTitle, pageTitle, children }) => {
+export const MainPageWithScroll: React.FC<MainPageWithScrollProps> = ({
+  headTitle,
+  pageTitle,
+  noBottomPadding,
+  children,
+}) => {
   return (
     <div className={styles.page}>
       <div>
         {headTitle}
         {pageTitle}
       </div>
       <div className={styles.contentContainer}>
-        <div className={classNames(styles.content, { [styles.cloud]: isCloudApp() })}>{children}</div>
+        <div
+          className={classNames(styles.content, {
+            [styles.cloud]: isCloudApp(),
```

Comment:
I started working on this but it's not as simple because in cloud, the intercom hook depends on having a context enabled, which depends on having intercom installed, so it involves some refactoring. What do you think about having an other function such as `hasIntercom()` that detects if `window.Intercom` is defined?

Reply:
I'm not sure you need the intercom hook to achieve this - just `useFeatureService()`. Something like this:

```jsx
const isIntercomEnabled = useFeature(FeatureItem.IntercomChat);

return (
        <div
          className={classNames(styles.content, {
            [styles.cloud]: isIntercomEnabled
          })}>
)
```

Then just put `FeatureItem.IntercomChat` in `defaultCloudFeatures` here: https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/hooks/services/Feature/constants.ts#L12

Reply:
That part is easy. If we're going to support intercom from the feature service, the app needs to be updated only to enable intercom when the feature is available. The issue is that the current cloud code assumes that intercom is always available, so some changes need to be made there to wrap around the `react-use-intercom` library.